hypothesis-python/tests/cover/test_regex.py

```diff
@@ -131,10 +131,10 @@ def test_can_generate(pattern, encode):
 
 
 @pytest.mark.parametrize('pattern', [
```

Comment:
Why did these change in this PR?

Reply:
Because two regex-related tests kept failing (thanks, example database!) and I didn't want to do the whole dance of fixing it in another PR and rebasing a million times.  Nothing more principled than that.